### PR TITLE
addition of e and f

### LINKED ISSUE #171: Selected works
How do we deal with Selected Works of an author, especially if he or she writes all kinds of different texts?
Shall we add another Genre called "Selected Works" and no Genre_type?

### LINKED ISSUE #175: trsl by pm
When we say that the note field is reserved for internal communication only, and we put the information "Translated by project member" into that field, we say that this piece of information should not be seen by the outer world, right? In a way I think this is a little wrong.

### PR DESCRIPTION
**Pull request by inamoto4**
Addition of e and f. The other entries of e and f from our old Excel sheet are not relevant to our database, except for all those stemming from Xiao Xiao 1999. This list should be dealt with in one go.

close #175
close #171

I allowed myself to go for the good, old data transfer from bad to good to check for possible final structural changes. Please note that no few of the entries in the Excel file describe reading acts which happened (long) before the CR. One example would be Zhou Ziren. We have to doublecheck everything before allowing it into our new, shiny database!

### CODE REVIEW

**Comment by duncdrum** [comment]
please fix conflicts

File: data/Genre.csv [hunk 1]
```diff
@@ -8,7 +8,7 @@ G06,shouchaoben fiction,en,fiction,2017-07-03,LH,2017-07-03,LH
 G07,science fiction,en,fiction,2017-07-03,LH,2017-07-03,LH
 G08,fairy tale,en,fiction,2017-07-03,LH,2017-07-03,LH
 G09,chuanqi,zh-Latn-pinyin,fiction,2017-07-03,LH,2017-07-03,LH
-G10,poetry,en,poetry,2017-07-03,LH,2017-07-03,LH
+,poetry,en,poetry,2017-07-03,LH,2017-07-03,LH
```
Comment (duncdrum) [inline data/Genre.csv]: missing ID

File: data/Person.csv [hunk 2]
```diff
@@ -164,8 +164,8 @@ P0082,Andreyev,Leonid,en,male,NULL,NULL,NULL,NULL,1871,1919,,,PL0030,,,https://e
 P0082,安德列耶夫,列昂尼德,zh,male,NULL,NULL,NULL,NULL,1871,1919,,,PL0030,,,https://en.wikipedia.org/wiki/Leonid_Andreyev,,,,,,2017-07-09,OS,2018-01-08,MG
 P0082,Андре́ев,Леони́д,ru,male,NULL,NULL,NULL,NULL,1871,1919,,,PL0030,,,https://en.wikipedia.org/wiki/Leonid_Andreyev,,,,,,2017-07-09,OS,2018-01-08,MG
 P0083,Anonymous,,en,,NULL,NULL,NULL,NULL,,,,,,,,,,,,,,2017-07-09,OS,2017-07-12,OS
-P0084,Aristotle,,en,male,NULL,NULL,NULL,NULL,0384,0322,,,PL0031,,,https://en.wikipedia.org/wiki/Aristotle,,,,,,2017-07-09,OS,2018-01-16,WH
-P0084,Ἀριστοτέλης,,grc,male,NULL,NULL,NULL,NULL,0384,0322,,,PL0031,,,https://en.wikipedia.org/wiki/Aristotle,,,,,,2017-07-09,OS,2018-01-16,WH
+P0084,Aristotle,,en,male,NULL,NULL,NULL,NULL,384,322,,,PL0031,,,https://en.wikipedia.org/wiki/Aristotle,,,,,,2017-07-09,OS,2018-01-16,WH
```
Comment (duncdrum) [inline data/Person.csv]: disable auto-correct

File: data/Person.csv [hunk 3]
```diff
@@ -402,3 +402,40 @@ P0217,Wang,Xiaoni,en,female,,,,,1955,,NULL,NULL,PL0093,,unknown,https://en.wikip
 P0217,王,小妮,zh,female,,,,,1955,,NULL,NULL,PL0093,,unknown,https://en.wikipedia.org/wiki/Wang_Xiaoni,,,,,,2018-01-12,MG,2018-01-16,MG
 P0218,Tang,Xiaodu,en,male,,,,,1954,,NULL,NULL,PL0116,,unknown,http://thescriptroad.org/guest/tang-xiaodu/,,,,,,2018-01-12,MG,2018-01-16,MG
 P0218,唐,小渡,zh,male,,,,,1954,,NULL,NULL,PL0116,,unknown,http://thescriptroad.org/guest/tang-xiaodu/,,,,,,2018-01-12,MG,2018-01-16,MG
+,,,,,,,,,,,,,,,,,,,,,,,,,
```
Comment (duncdrum) [inline data/Person.csv]: why add empty rows?

File: data/Text.csv [hunk 4]
```diff
@@ -249,32 +249,96 @@ T00147,,,en,P0209,,,2011-09-13,Remembrance,,NULL,http://www.xujuneberlein.com/re
...
 T00161,七十年代记忆片段,,zh,P0217,G03,PL0002,2009,Sanlian shudian,,NULL,T00044,NULL,457-474,,,2018-01-12,MG,2018-01-12,MG
-T00162,1976,My First Love Died like Shabby Cotton,en,P0218,G03,PL0002,2009,Sanlian shudian,,NULL,T00044,NULL,475-492,,Translated by project member,2018-01-12,MG,2018-01-12,MG
+T00162,1976,My First Love Died like Shabby Cotton,en,P0218,G03,PL0002,2009,Sanlian shudian,,NULL,T00044,NULL,475-492,Translated by project member,,2018-01-12,MG,2018-01-12,MG
 T00162,1976,初恋败絮,zh,P0218,G03,PL0002,2009,Sanlian shudian,,NULL,T00044,NULL,475-492,,,2018-01-12,MG,2018-01-12,MG
-T00163,The Black Painting Crisis,,en,P0137,G03,PL0002,2009,Sanlian shudian,,NULL,T00044,NULL,349-358,,Translated by project member,2018-01-12,MG,2018-01-12,MG
+T00163,The Black Painting Crisis,,en,P0137,G03,PL0002,2009,Sanlian shudian,,NULL,T00044,NULL,349-358,Translated by project member,,2018-01-12,MG,2018-01-12,MG
 T00163,黑画风波,,zh,P0137,G03,PL0002,2009,Sanlian shudian,,NULL,T00044,NULL,349-358,,,2018-01-12,MG,2018-01-12,MG
+,,,,,,,,,,,,,,,,,,,
```
Comment (duncdrum) [inline data/Text.csv]: s.a.

File: data/Text.csv [hunk 5]
```diff
@@ -249,32 +249,96 @@ T00147,,,en,P0209,,,2011-09-13,Remembrance,,NULL,http://www.xujuneberlein.com/re
...
+T00188,中国通史简编,,zh,P0254,G19,,,,,,,,,,,2018-01-25,OS,2018-01-25,OS
+T00189,Broken chapter,,en,P0009,G03,,,,,,T00044,,31-50,Translated by project member,,2018-01-25,OS,2018-01-25,OS
+T00189,断章,,zh,P0009,G03,,,,,,T00044,,31-50,,,2018-01-25,OS,2018-01-25,OS
+T00190,The unforgettable 1971,,en,P0219,G03,,,,,,T00044,,257-278 ,Translated by project member,,2018-01-25,OS,2018-01-25,OS
+T00190,难忘的1971,,zh,P0219,G03,,,,,,T00044,,257-278 ,,,2018-01-25,OS,2018-01-25,OS
+T00191,Stories Old and New,,en,P0192,G05,PL0002,1958,Renmin wenxue chubanshe,1620,no,,,642,,,2018-01-25,OS,2018-01-25,OS
+T00191,古今小说,,zh,P0192,G05,PL0002,1958,Renmin wenxue chubanshe,1620,no,,,642,,,2018-01-25,OS,2018-01-25,OS
+T00192,The unjustly murdered woman from the neighbourhood,,en,P0220,G03,,2010,,2010,NULL,http://www.xujuneberlein.com/rem57.html ,57,ONLINE SOURCE,,,2018-01-25,OS,2018-01-25,OS
+T00192,被冤杀的街坊大妈 ,,zh,P0220,G03,,2010,,2010,NULL,http://www.xujuneberlein.com/rem57.html ,57,ONLINE SOURCE,,,2018-01-25,OS,2018-01-25,OS
+,,,en,P0033,,,,,,,,,,,,,,,
```
Comment (duncdrum) [inline data/Text.csv]: bad format

**Comment by duncdrum** [comment]
please open a new PR with properly formated csv

**Comment by inamoto4** [comment]
Thx for the corrections. I will again try to solve the autocorrect problems, it did not work last time. 

e and f are obviously the names of the old Excel table, which is sorted alphabetically. zz is Zhou Ziren, which I have mentioned in my commentary. The blank lines were left to insert the letters c and d, as it would be nice to have them in an order. 
Yes, we can filter everything out, but so far it takes such a long time to fill in one single entry - whether it is a fully new one or just one to be transferred from the old table -, that we really need to think about what is economical and what is not. And in it does not make sense to include the 1950s in the realm of our research project with our limited resources. But you are free to change the focus in the ERC time. :-)

**Pull request merged**